Source/SIMPLib/DataContainers/DataContainerProxy.h

```diff
@@ -193,7 +201,7 @@ class SIMPLib_EXPORT DataContainerProxy
    * @param the name of the attribute matrix
    * @return the attribute matrix proxy
    */
-  AttributeMatrixProxy& getAttributeMatrixProxy(const QString& name);
+  AttributeMatrixProxy& getAttributeMatrixProxy(const QString& m_Name);
```

Comment:
Are you sure about renaming the parameters in setFlag and getAttributeMatrixProxy?  If not, you might want to update the insertAttributeMatrix declaration as well.

Reply:
Same with the flag parameter in setFlags below.